src/sonic-host-services/scripts/caclmgrd

```diff
@@ -277,19 +277,24 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
 
     def get_chassis_midplane_interface_ip(self):
 
+        chassis_midplane_dev_name_command = "ip -4 -o addr show " + "eth1-midplane" +\
+                                             " | awk '{print $0}' | cut -d' ' -f2"
+
+        midplane_dev_name = self.run_commands([chassis_midplane_dev_name_command])
+
         chassis_midplane_ip_command = "ip -4 -o addr show " + "eth1-midplane" +\
                                  " | awk '{print $4}' | cut -d'/' -f1 | head -1"
-        return self.run_commands([chassis_midplane_ip_command])
-
+        midplane_ip = self.run_commands([chassis_midplane_ip_command])
+        return midplane_dev_name, midplane_ip
```

Comment:
i am curious why we do not use this api?

https://github.com/sonic-net/sonic-platform-common/blob/master/sonic_platform_base/module_base.py#L537